### PR TITLE
basic loop of data being displayed, with some css

### PR DESCRIPTION
**Pull request by Dave-Turnbull**
forgot to make a new branch for the feature

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,53 @@
-import logo from './logo.svg';
-import './App.css';
+// src/App.js
```
Comment (olivershooter) [inline src/App.js:1]: Top level comments should normally be large and clear:

`/*

  APP.JS

*/`

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,53 @@
...
+import React from "react";
+import "./App.css";
+import Litrature from "./litrature";
```
Comment (olivershooter) [inline src/App.js:4]: Literature or litrature?

File: src/App.js [hunk 3]
```diff
@@ -1,23 +1,53 @@
...
+
+const dummyData = [
```
Comment (olivershooter) [inline src/App.js:6]: Would like to refactor this, logic should always go in seperate files and never top level in the app.js. We also probably just want to make a JSON file with this info and add it to the utils folder.

File: src/App.js [hunk 4]
```diff
@@ -1,23 +1,53 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h1>Litrature</h1>
+      {dummyData.map((litrature) => (
+        <Litrature
```
Comment (olivershooter) [inline src/App.js:41]: Nice use of components!

File: src/index.css [hunk 5]
```diff
@@ -1,13 +1,71 @@
...
+  padding: 20px;
+  width: 60%;
+  margin: 20px auto;
+  position: relative; /* Add this line to make sure the pseudo-element is positioned relative to the .litrature container */
+  min-height: 100px;
+  color: rgba(0, 0, 0, 0.834);
+}
+
+.litrature:before {
+  content: ""; /* Add this line to create the pseudo-element */
```
Comment (olivershooter) [inline src/index.css:29]: Don't mind some comments in here as its the CSS file, long as they're minimal (same for the JS files really, just need to keep it minimal as code should do the speaking)

File: src/litrature.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
+// src/Litrature.js
+import React from "react";
+
+const Litrature = ({
```
Comment (olivershooter) [inline src/litrature.js:4]: Maybe not the right name for the file but for no doesn't matter. Normally name the thing they: "DummyData.js".

Also needs to go into a component folder. src/components/Literature/index.js

**Review by olivershooter — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (olivershooter) [reply src/App.js:1]: By top level I mean, comments that name the file to show what it is.

Comment (Dave-Turnbull) [reply src/App.js:1]: forgot to change this one

File: src/App.js (hunk 2 above)
Comment (Dave-Turnbull) [reply src/App.js]: im bad at spelling

File: src/App.js (hunk 3 above)
Comment (Dave-Turnbull) [reply src/App.js]: i had to look up what refactor is, dummy data is in a json now

File: src/App.js (hunk 4 above)
Comment (Dave-Turnbull) [reply src/App.js]: thanks chatGPT!

File: src/litrature.js (hunk 6 above)
Comment (Dave-Turnbull) [reply src/litrature.js]: added to a component folder

**Review by olivershooter — DISMISSED** [review]

**Comment by Dave-Turnbull** [comment]
hello

**Review by Dave-Turnbull — COMMENTED** [review]
great work me!

File: src/App.js [hunk 7]
```diff
@@ -1,6 +1,11 @@
 /*==========
-  SRC/APP.JS
```
Comment (Dave-Turnbull) [inline src/App.js]: just testing commenting

**Review by Dave-Turnbull — COMMENTED** [review]
10/10 code

**Review by olivershooter — APPROVED** [review]

**Pull request merged**